### ISSUE TITLE
Meta description property needed for all pages

### ISSUE THREAD

**Issue by clairernovotny** [dotnet-foundation/website#31]
Every page should have a meta description element that would get output. This is important for SEO and should probably be a standard part of Statiq.Web

**Comment by daveaglick** [comment]
Good idea - it's likely up to the theme to add this, so I'll put it in the master layout file. Is the intent to hand-craft a short description for each page an add that as front matter like `MetaDescription: This page blah blah blah` or are you hoping it'll be automatic?

We do get an excerpt automatically which is the first HTML `<p>` or can be overridden in front matter, but that generally contains markup and the first paragraph probably isn't quite appropriate for an overall description of a page.

TL;DR: where should the content for this description come from?

**Comment by clairernovotny** [comment]
I believe it should always have something, so maybe a default excerpt that can be overridden by the value?

**Comment by clairernovotny** [comment]
Other things that may be in a similar category: social media twitter cards, etc. like they may have a site-wide default, but maybe on a blog it should default to the featured image? And then overridden on a particular post?
